sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -141,9 +161,12 @@ def __init__(
         self.primary_user_assigned_identity = primary_user_assigned_identity
         self.managed_network = managed_network
         self.enable_data_isolation = enable_data_isolation
-        self.workspace_hub = workspace_hub
```

Comment:
You had a good point in the bug bash call. Let's make sure this is a change allowed by Azure SDK team.

Reply:
I read over the bug bash chat but I don't see the issue that was mentioned. What's the problem here?